### ISSUE TITLE
Crash with strings that may look like a scientific notation

### ISSUE THREAD

**Issue by estebarb** [thombashi/sqlitebiter#70]
When the table has values that are strings but look like scientific notation, like `45e76582` sqlitebitter crashes. For example, use the following HTML without --no-type-inference flag:

```html
<table>
    <caption>example</caption>
  <tr>
    <th>value</th>
  </tr>
  <tr>
    <td>45e76582</td>
  </tr>
</table>
```

It also crashes when using the flag `--no-type-inference`, but wasn't able to create a minimal example for this crash.

**Comment by thombashi** [comment]
@estebarb 
Thank you for your report.

I had fixed the problem at `sqlitebiter 0.28.1`.
Please try to upgrade and re-execute.

>It also crashes when using the flag --no-type-inference, 

I could not reproduce this case, please let me know if you find out how to reproduce this case.

**Comment by thombashi** [comment]
I'll close the issue.
Feel free to reopen if you still have any problems about the issue.
